os/net/socket/net_poll.c

```diff
@@ -0,0 +1,46 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#ifdef CONFIG_NET
+
+#include <tinyara/net/net.h>
+#ifdef CONFIG_NET_LWIP
```

Comment:
if `CONFIG_NET_LWIP` goes along with `CONFIG_NET` then maybe this check is not necessary. Please ignore if not the case.

Reply:
if different network library is used then this option should be neccessary.